### DISCUSSION TITLE
Give me stars and i'll acpt your answer (for badges)

### DISCUSSION THREAD

**Discussion by marv1n-le** [recodehive/Stackoverflow-Analysis#481] [Q&A] [answered]
https://github.com/marv1n-le/recipe-haven
give me star please

**Comment by ducanhismyname** [comment]
okay

**Comment by damoves005** [comment, marked as answer]
I already click the star on your repo. Mark my comment as an answer too to get the  Galaxy Brain badge. Thank you and appreciate it.

**Comment by ghost** [comment]
Bro Please give me a star to achieve the badge
https://github.com/mte-tonmoy/mte-tonmoy
&  I already give you star on your repository.

**Comment by nandkishor-vasi** [comment]
I've already starred the repo! 🙌
If possible, please consider marking my comment as the accepted answer so I can earn the badge.
Thanks a lot really appreciate your help!
